store/assets/currencies.go

```diff
@@ -0,0 +1,46 @@
+package assets
+
+import (
+	"fmt"
+	"math/big"
+)
+
+// Link contains a field to represent the smallest units of LINK
+type Link big.Int
```

Comment:
Just for clarity and to put my pedant hat on, this is actually a new named type, not a type alias.

[Type alias](https://github.com/golang/proposal/blob/master/design/18130-type-alias.md#proposal)
```golang
type NewAlias = big.Int // type alias
type NewType big.Int // new named type
```

This is why you had to redefine `SetInt64` below.